### PR TITLE
Include dates for matches and seasons

### PR DESCRIPTION
**Pull request by nilssonemil**
This allows to determine which is the current active season and upcoming
matches for example.

### CODE REVIEW

File: resources/public/openapi/spec.yml [hunk 1]
```diff
@@ -512,6 +512,9 @@ components:
           type: integer
         referee:
           type: integer
+        match-date:
```
Comment (hansenemily) [inline resources/public/openapi/spec.yml:515]: I guess that this match date also includes the time? It seems a bit misleading to name it match-date in the API spec.. Maybe rename to something lite match-datetime or match-time?

**Review by hansenemily — CHANGES_REQUESTED** [review]
Will you later add some kind of possibility to edit the time specifically? Something like what you have done for referee?

And you seem to have broken tests?

**Comment by nilssonemil** [comment]
> Will you later add some kind of possibility to edit the time specifically? Something like what you have done for referee?
> 
> And you seem to have broken tests?

Yes, later I will add functionality to update the dates.

The tests were already broken. But are fixed in master :small_airplane:

File: resources/public/openapi/spec.yml (hunk 1 above)
Comment (nilssonemil) [reply resources/public/openapi/spec.yml:515]: Personally, I like match-date, as I think both match-time and match-datetime sounds weird. Datetime for me is just a data type, and I would not likely include that in my variable names etc.

I see your point, but I also disagree. I think I would not be surprised if I saw time to be part of a date object. Just look at [java.util.Date](https://docs.oracle.com/javase/8/docs/api/?java/util/Date.html).

However, maybe a description for match-date would be nice in the API spec.

Comment (hansenemily) [reply resources/public/openapi/spec.yml:515]: I agree with you on datetime, but not regarding date vs time. But it’s really not a big issue, so if you add a description then I will be fine with that!

**Review by hansenemily — APPROVED** [review]

File: resources/public/openapi/spec.yml (hunk 1 above)
Comment (hansenemily) [reply resources/public/openapi/spec.yml:515]: @nilssonemil I know this has already been merged, but I thought of a better name for the variable, that might solve both yours and my concerns: match-start? What do you think?

**Pull request merged**